qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -386,7 +386,17 @@ def parse_offset_get(s):
                            get_cmd='OUTP? 4',
                            get_parser=float,
                            unit='deg')
-
+        
+        self.add_function('snap_xy',
```

Comment:
I meant class bound method, like `def snap_xy(...): ....`. Sorry that I was not clear enough.